autoload/neomake/utils.vim

```diff
@@ -409,3 +409,22 @@ endfunction
 function! neomake#utils#path_sep() abort
     return neomake#utils#IsRunningWindows() ? ';' : ':'
 endfunction
+
+" Find a file by going up the directories form the start directory
```

Comment:
Add a reference to the Syntastic function it's based on?!
https://github.com/vim-syntastic/syntastic/blob/2124af80abfb856e8806c3b508b3982b7157c4a3/autoload/syntastic/util.vim#L261-L297.